### DISCUSSION TITLE
Weather Widget/Flyout Looks Weird

### DISCUSSION THREAD

**Discussion by Thaumatarge** [valinet/ExplorerPatcher#780] [Q&A] [answered]
Hello, I have recently solved a couple issues by restoring default settings and reinstalling the program (Mainly something to do with memory adresses whenever I shut down my computer, and the inability to update the program properly), however one issue persisted involving the weather flyout.
I cannot interact with the flyout, and after reinstalling the program it has this weird white border around it. This is the first time this has happened to me. Any ideas on how I could fix it?
![image](https://user-images.githubusercontent.com/34694089/152542637-642e4a9e-a4f6-42b2-8586-03b021f8f394.png)

**Comment by valinet** [comment, marked as answer]
Disable weather widget in Properties, go to `%APPDATA%\ExplorerPatcher` and remove the `ep_weather_host` folder, reenable the widget. It's because I rolled back the latest pre-release (42.4) because it was buggy, but I did not have and still do not have time to fix it, so in the meantime, reinstalling will get you pre-release 42.3 which rendered the widget a bit differently. As I said, pre-releases may be buggy, things like this can happen, this week was full and I did not have even a second worth of time to work on this. I eventually plan to fix it properly, but I can't do it right now.

Thanks

P.S. That overlay appears because the browser is displaying that stupid GDPR consent pop-up that Google has.

**Reply by Thaumatarge** [reply]
Thanks! I am aware that pre-releases will be buggy, it was just an odd thing that annoyed me and wondered if it was an actual issue that was good to notify of/that wasn't known, or if it was just a small bug. Take your time, you are doing god's work with this little program and thanks again!
